sql_pracownicy.sql

```diff
@@ -0,0 +1,112 @@
+CREATE DATABASE myemployees COLLATE utf8mb4_polish_ci;
+
+#1
+CREATE TABLE employee (
+	id INT PRIMARY KEY AUTO_INCREMENT,
+    first_name VARCHAR(20) NOT NULL,
+    last_name VARCHAR(20) NOT NULL,
+    salary INT NOT NULL,
+    birth_date DATE NOT NULL,
+    position VARCHAR(30) NOT NULL
+);
+
+#2
+INSERT INTO employee 
+	(first_name, last_name, salary, birth_date, position)
+VALUES
+	('Alicja', 'Nowak', 2050, '1992-04-02', 'sales'),
+	('Tatiana', 'Fredzel', 3500, '1991-08-08', 'engineer'),
+	('Boguslaw', 'Palec', 5000, '1982-10-10', 'manager'),
+	('Katarzyna', 'Madrala', 7000, '1956-09-08', 'director'),
+	('Piotr', 'Truc', 5600, '1998-02-01', 'lawyer'),
+	('Milena', 'Gotowa', 6300, '1977-02-01', 'accountant');
+    
+#3
+select * from employee ORDER BY last_name;
+
+#4
+select * from employee where position = 'director';
+
+###5
+select * from employee where extract(year from curdate()) - extract( year from birth_date) >= 30;
+
+#6
+select first_name, last_name, salary, birth_date, position, salary * 1.1 as salary_with_raise from employee where position = 'director';
+
+###7
+select * from employee ORDER BY birth_date desc limit 1;
```

Comment:
To pobierze tylko jednego pracownika. Rozwiązaniem jest utworzenie subquery dla WHERE i powrównać datę urodzenia do najmniejszej